### DISCUSSION TITLE
backwards compatibility with stage damage compute

### DISCUSSION THREAD

**Discussion by codymccoy** [HydrologicEngineeringCenter/HEC-FDA#569] [General]
The hydraulic enums changed from these:
![image](https://user-images.githubusercontent.com/47757848/194100670-0cf3b916-ab1b-4c4b-a612-f3ee2b7a17bf.png)

to these:
![image](https://user-images.githubusercontent.com/47757848/194100832-8648eb7c-1f79-4f34-b35b-a415cd96c0b8.png)

either delete your hydros from your study before getting the latest code, or update the values directly in the xml.

**Comment by rnugent3** [comment]
This applies to studies created prior to October 2022.
